Purchases/Identity/CustomerInfoManager.swift

```diff
@@ -174,9 +166,35 @@ class CustomerInfoManager {
         }
     }
 
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+    var customerInfoStream: AsyncStream<CustomerInfo> {
+        return AsyncStream(bufferingPolicy: .bufferingNewest(1)) { continuation in
+            let disposable = self.monitorChanges { continuation.yield($0) }
+
+            continuation.onTermination = { @Sendable _ in disposable() }
+        }
+    }
+
+    private var customerInfoChanges: [Int: (CustomerInfo) -> Void] = [:]
```

Comment:
i find this naming a little confusing...these are actually callbacks, or "monitors", right? or am i missing something?

Reply:
Yeah it's a bad name. I changed it to `customerInfoObservers`..